packages/fx-core/src/component/generator/apiSpec/helper.ts

```diff
@@ -1544,12 +1568,22 @@ async function updatePromptSuggestions(specItems: SpecObject[], manifestPath: st
     manifest.bots![0].commandLists = [
       {
         scopes: ["personal"],
-        commands: descriptions.map((des) => {
-          return {
-            title: des.slice(0, 32),
-            description: des.slice(0, 128),
-          };
-        }),
+        commands: [
+          {
+            title: "Hello, how can you help me?",
```

Comment:
minor: I just realized that the template generates its own command. Let's sync with @MuyangAmigo to determine whether we still want to add these fixed commands.
It's not blocking, so feel free to go ahead and merge first.